package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "5.13.1",
+  "version": "5.14.0",
```

Comment:
Do we need a minor version this week? It seems we could release a new patch version

Reply:
The reson is because of the exported `GridBooleanCell` component. I'm just folowing semver.
I don't think we need to hold back on doing minor version releases as long as we keep the nonbreaking change policy.